docs/conf.py

```diff
@@ -303,11 +305,13 @@
 # -- Options for linkcheck --
 
 linkcheck_retries = 3
+linkcheck_workers = 32
 
 linkcheck_ignore = [
-  r'http://127.0.0.1(:\d+)?/?',
-  r'http://localhost(:\d+)?/?',
-  'https://forum.securedrop.org/admin/users/list/active',
-  'https://weblate.securedrop.org/projects/securedrop/securedrop/#repository',
-  'https://github.com/freedomofpress/securedrop-debian-packages-lfs',
+    r"http://127.0.0.1(:\d+)?/?",
+    r"http://localhost(:\d+)?/?",
+    "https://forum.securedrop.org/admin/users/list/active",
+    "https://weblate.securedrop.org/projects/securedrop/securedrop/#repository",
+    "https://github.com/freedomofpress/securedrop-debian-packages-lfs",
+    r"https://github.com/freedomofpress/securedrop/issues/.*",
```

Comment:
No 429s for issues tickets, but I did see one for a path on the develop branch:
```
Warning, treated as error:
/home/user/securedrop/docs/development/tips_and_tricks.rst.rst:134:broken link: https://github.com/freedomofpress/securedrop/tree/develop/docs/diagrams (429 Client Error: too many requests for url: https://github.com/freedomofpress/securedrop/tree/develop/docs/diagrams)
```
It might be worth adding `https://github.com/freedomofpress/securedrop/tree/.*` to the list.

Reply:
Done.

Reply:
Looks good, passing for me locally now